apps/ex_wire/lib/ex_wire/p2p/server.ex

```diff
@@ -110,7 +110,7 @@ defmodule ExWire.P2P.Server do
   """
   @spec get_peer(pid()) :: Peer.t()
   def get_peer(pid) do
-    GenServer.call(pid, :get_peer)
+    GenServer.call(pid, :get_peer, :infinity)
```

Comment:
Yes, right now, there's really no reason to timeout here. Though, this might be mitigated by better arrangement of servers. But again, the timeout wasn't really helping us.